src/common-components/account-short-entry.jsx

```diff
@@ -153,7 +153,7 @@ const delayRandomBase = Math.random() * 400;
 /**
  * @param {AccountInfo} account
  */
-function getAvatarDelay(account) {
+export function getAvatarDelay(account) {
```

Comment:
I think we can get rid of this change too, since the place it was being exported for also ended up being unused?